base/ca/src/main/java/com/netscape/ca/CertificateAuthority.java

```diff
@@ -1557,28 +1558,35 @@ private BasicOCSPResponse sign(ResponseData rd) throws EBaseException {
         }
     }
 
-    private SingleResponse processRequest(Request req) {
+    public SingleResponse processRequest(Request req) {
 
+        String name = "CertificateAuthority: processRequest: ";
         CAEngine engine = CAEngine.getInstance();
         CertificateRepository certificateRepository = engine.getCertificateRepository();
 
+        X509CertImpl caCert = mSigningUnit.getCertImpl();
+        X509Key key = (X509Key) caCert.getPublicKey();
+
         CertID cid = req.getCertID();
         INTEGER serialNo = cid.getSerialNumber();
-        logger.debug("CertificateAuthority: processing request for cert 0x" + serialNo.toString(16));
+        logger.debug( name + "for cert 0x" + serialNo.toString(16));
 
         CertStatus certStatus = null;
         GeneralizedTime thisUpdate = new GeneralizedTime(new Date());
 
         byte[] nameHash = null;
+        byte[] keyHash = null;
         String digestName = cid.getDigestName();
         if (digestName != null) {
             try {
                 MessageDigest md = MessageDigest.getInstance(digestName);
                 nameHash = md.digest(mName.getEncoded());
+                keyHash = md.digest(key.getKey());
             } catch (NoSuchAlgorithmException | IOException e) {
             }
```

Comment:
Looks like we've been ignoring these types of errors, but at least let's generate the same log message here.